src/dll/Hooks/ValidateScripts.cpp

```diff
@@ -89,3 +89,59 @@ bool Hooks::ValidateScripts::Detach()
     isAttached = result != NO_ERROR;
     return !isAttached;
 }
+
+std::wstring WritePopupMessage(const std::vector<ValidationError>& validationErrors,
+                               const std::vector<PluginSystem::PluginName>& incompatiblePlugins)
+{
+    if (validationErrors.empty())
+    {
+        return {};
+    }
+
+    std::wstring message;
```

Comment:
```suggestion
    fmt::wmemory_buffer message;
```

Reply:
I think you dont have to use std::back_inserter with this but am unsure :suspect:

Reply:
I mean with what Wopss wanted you to replace this with... Not with what you had (additional possible change I suppose)